e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -130,7 +130,7 @@ $certificateHashAlgorithm = "SHA256"
 
 if ($GenerateResourcesForDevOpsPipeline)
 {
-    $iothubUnitsToBeCreated = 3
+    $iothubUnitsToBeCreated = 3;
```

Comment:
Consistent use of semi-colons.

Avoided using them when calling az cli commands or powershell functions.

Used it when setting variables, calling .NET functions, and where actually required (maps).